pkg/crossplane/application.go

```diff
@@ -0,0 +1,79 @@
+package crossplane
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var (
+	fieldsAppDesiredRes          = append(fieldsStatus, "desiredResources")
+	fieldsAppSubmittedRes        = append(fieldsStatus, "submittedResources")
+	fieldsAppResourceSelector    = append(fieldsSpec, "resourceSelector")
+	fieldsAppResourceMatchLabels = append(fieldsAppResourceSelector, "matchLabels")
+
+	stateAppSubmitted = "Submitted"
+)
+
+type Application struct {
+	u *unstructured.Unstructured
```

Comment:
In general I like descriptive variable names over single-letter names, but if this is the go idiom for untyped and anonymous variables like this, I'm okay with it.

Reply:
Go idiom likes short names but not sure this applies here. 
Changed it as u -> instance which is already something that I used somewhere else in the code.